app/assets/stylesheets/application.css.scss

```diff
@@ -34,3 +34,12 @@ body {
 .theme-showcase > p > .btn {
   margin: 5px 0;
 }
+
+.menu {
+  position: relative;
+}
+
+path {
+  stroke: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered fill-rule, stroke